int32[3] item_num_chosen_by_pad
bool[3] is_item_available
int32[3] robot_service_sequence
